src/objects.js

```diff
@@ -1,41 +1,65 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {
+    name: name,
+    age: age
+  }
+
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
+
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  return object.hasOwnProperty(property);
```

Comment:
there is an eslint rule forbidding the use of built in methods, such as `hasOwnProperty`. Instead, you could write this as `return object[property] !== undefined;`